src/pages/FAQ.tsx

```diff
@@ -54,30 +60,74 @@ export default function FAQ() {
         </div>
 
         <div className="flex flex-col items-center max-w-3xl px-4">
-          <ul className="list-none py-2 space-y-12">
+          <div className="py-2 space-y-3">
             {searchResults.map((FAQ, i) => {
               const { question, answer } = FAQ;
               return (
-                <li className="space-y-4" key={`q-${i}`}>
-                  <div className="font-bold text-lg">
-                    <strong>{question}</strong>
+                <div key={`q-${i}`} className="">
+                  <h2 className={`py-3 px-3 bg-[#233B49] ${activeIndex === i? 'delay-100 rounded-t-xl' : 'delay-100 rounded-xl' }`}>
+                    <button
+                      id="faqs-title-07"
+                      type="button"
+                      className="flex items-center justify-between w-full text-left font-semibold py-2"
+                      onClick={() => toggleAccordion(i)}
+                      // aria-expanded={expanded}
+                      aria-controls="faqs-text-07"
+                    >
+                      <span>{question}</span>
+                      <svg
+                        className={`fill-white relative z-[0] shrink-0 ml-8 transform origin-center transition duration-200 ease-out ${
+                          activeIndex === i ? "rotate-180" : ""
+                        }`}
+                        width="16"
+                        height="16"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <rect
```

Comment:
For this, is it possible to use [react icons](https://react-icons.github.io/react-icons/) instead of a custom svg?

Reply:
> For this, is it possible to use [react icons](https://react-icons.github.io/react-icons/) instead of a custom svg?

yes, i used two icons, in front of each other, and on clicking their scale is being changed